docs/geedocs/5.2.1/answer/7160000.html

```diff
@@ -23,35 +24,35 @@
 <a name="top_of_file"></a>
 <p><img src="../art/common/googlelogo_color_260x88dp.png" width="130" height="44" alt="Google logo" /></p>
 <h1><a href="../index.html">Google Earth Enterprise Documentation Home</a> | Release notes</h1>
-<h2>Release notes: Open GEE 5.2.1</h2>
-<p>Open GEE 5.2.1 the second open source release of Google Earth Enterprise.  It is designed to be compatible with GEE 5.1.3 with the exception of MrSID support.
-Open GEE 5.2.1 includes a number of code and library changes to support the open source effort.</p>
+<h2>Release notes: Open GEE 5.2.0</h2>
+<p>Open GEE 5.2.0 is the first open source release of Google Earth Enterprise.  It is designed to be compatible with GEE 5.1.3 with the exception of MrSID support.
```

Comment:
Very minor: the 'is' near the beginning of this sentence is missing from 5.2.0/answers/7160000.html.  That doesn't have anything to do with this PR, but you could fix it if you like.